frontend/src/app/modules/boards/new-board-modal/new-board-modal.component.ts

```diff
@@ -101,5 +120,37 @@ export class NewBoardModalComponent extends OpModalComponent {
         this.halNotification.handleRawError(error);
       });
   }
+  private addDescription(tiles:ITileViewEntry[]) {
+    tiles.forEach(element => {
+      if (element.attribute !== 'basic') {
+      element.description = this.I18n.t('js.boards.board_type.action_text',
+      { attribute: this.I18n.t('js.boards.board_type.action_type.' + element.attribute )}); }
+    });
+  }
+  private addIcon(tiles:ITileViewEntry[]) {
+    tiles.forEach(element => {
+      if (element.attribute === 'assignee') {
```

Comment:
I would suggest to move these icons into the *ActionService classes themselves. Add a property in the baseclass (similar to filterName here: https://github.com/opf/openproject/blob/dev/frontend/src/app/modules/boards/board/board-actions/board-action.service.ts#L43) This would allow to keep the definition of name/description and icon in the service? What do you think?